Luminary099/R30.agc

```diff
@@ -453,10 +454,10 @@ SKIPTPER	STODL	-TPER
 
```

Comment:
![image](https://user-images.githubusercontent.com/3440094/95390009-9b8dd600-08ec-11eb-890b-0cac0f6c7d81.png)
Line 437 (above):

- 2 extra spaces after `IF HPER L/`
- No space between `35)` and `KFT.`